code/controllers/configuration.dm

```diff
@@ -360,6 +361,8 @@
 					config.rulesurl = value
 				if("githuburl")
 					config.githuburl = value
+				if("githubrepoid")
+					config.githuburl = value

```

Comment:
convert to a number here, then check isnum below, its cheaper and how most things do it.